packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/ServiceExecutionState.ts

```diff
@@ -394,13 +394,20 @@ export abstract class ServiceExecutionContextState {
 }
 
 export class SingleExecutionContextState extends ServiceExecutionContextState {
-  declare executionContext: PureSingleExecution;
+  declare executionContext1: PureSingleExecution;
```

Comment:
why was this renamed?

Reply:
So this property comes from parent class and because now we change given mapping and runtime could be defined, so couldnt have the same name